### PR TITLE
[metal3] Dashboard fixes

### PR DESCRIPTION
**Pull request by honza**
Following up on some comments made in #2970

### CODE REVIEW

File: frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/EventsCard.tsx [hunk 1]
```diff
@@ -11,31 +11,18 @@ import ActivityBody, {
   Activity,
 } from '@console/shared/src/components/dashboard/activity-card/ActivityBody';
 import ActivityItem from '@console/shared/src/components/dashboard/activity-card/ActivityItem';
-import {
-  EventKind,
-  K8sResourceKind,
-  MachineKind,
-  referenceForModel,
-} from '@console/internal/module/k8s';
+import { EventKind, K8sResourceKind, MachineKind } from '@console/internal/module/k8s';
```
Comment (knowncitizen) [inline frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/EventsCard.tsx]: did prettier/eslint move this to one line?

Comment (honza) [reply frontend/packages/metal3-plugin/src/components/baremetal-hosts/dashboard/EventsCard.tsx]: Yes.

**Pull request merged**